grafana-plugin/src/pages/incident/Incident.helpers.tsx

```diff
@@ -13,36 +13,37 @@ import { User } from 'models/user/user.types';
 import SilenceDropdown from 'pages/incidents/parts/SilenceDropdown';
 import { move } from 'state/helpers';
 import { UserActions } from 'utils/authorization';
+import { COLOR_DANGER, COLOR_PRIMARY, COLOR_SECONDARY, COLOR_WARNING } from 'utils/consts';
 
 export function getIncidentStatusTag(alert: Alert) {
   switch (alert.status) {
     case IncidentStatus.New:
       return (
-        <Tag color="#E02F44">
+        <Tag color={COLOR_DANGER}>
```

Comment:
is it possible to pass string like `color="var(--color-danger)"`? and add `--color-danger` to vars.css as all other color consts

Reply:
Let's do this in a separate PR